webcomponent/download-openapi-jar.sh

```
@@ -72,16 +44,10 @@ if [ -f "$DIR/$LOCAL_JAR" ]; then
 echo " => Downloading .jar file:"
-echo "    URL:        $URL"
 echo "    Target-file: $DIR/$LOCAL_JAR"
 
-if command -v curl >/dev/null 2>&1; then
-  echo "  => Using curl"
-  curl -fSL "$URL" -o "$DIR/$LOCAL_JAR"
-else
-  echo "  => Using wget"
-  wget -O "$DIR/$LOCAL_JAR" "$URL"
-fi
+mvn dependency:copy -Dartifact=org.openapitools:openapi-generator-cli:$VER:jar -DoutputDirectory=.
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)